crates/llm-local/src/lib.rs

```diff
@@ -164,73 +144,36 @@ impl LocalLlmEngine {
     async fn inferencing_model(
         &mut self,
         model: wasi_llm::InferencingModel,
-    ) -> Result<Arc<dyn Model>, wasi_llm::Error> {
-        let use_gpu = self.use_gpu;
-        let progress_fn = |_| {};
-        let model = match self.inferencing_models.entry((model.clone(), use_gpu)) {
+    ) -> Result<Arc<dyn CachedInferencingModel>, wasi_llm::Error> {
+        // let use_gpu = self.use_gpu;
```

Comment:
What's happening here? Are we completely abandoning this option. If so, let's remove it all the way up the stack.

Reply:
Yes, we do not need it anymore.

Reply:
will remove it all the way